src/components/Draggable/Draggable.module.scss

```diff
@@ -18,6 +18,10 @@
   will-change: transform;
   z-index: 1;
 
+  &.preview {
+    transition: none !important;
```

Comment:
Is the `!important` necessary? If so, why?

Reply:
Works without, so I'll remove it